Add TcPdfOutput

sprain opened the pull request:
Resolves #50

* Integrates `TcPdfOutput` into the library (as currently available [as a separate project](https://github.com/sdespont/php-swiss-qr-bill-tcpdf))
* Improves setup of examples accordingly

**Todos**

- [x] Make sure tests pass reliably
- [x] Fix wrongly placed placeholders (see [this file for example](https://github.com/sprain/php-swiss-qr-bill/blob/49c59f0fc7d5414223e503c6ce78f78697fc06da/tests/TestData/TcPdfOutput/qr-additional-information.pdf))

sdespont commented:
Is it normal that some reference files have the placeholders **and** amounts ? Like qr-additional-information.pdf for example?

sdespont commented on src/PaymentPart/Output/TcPdfOutput/TcPdfOutput.php:
```diff
@@ -0,0 +1,324 @@
+
+            $lineLength = 3;
+
+            // Not implemented
```
In fact, it is implemented now :o)

sprain commented:
> Is it normal that some reference files have the placeholders and amounts ? 

It‘s a valid use-case and therefore sprinkled into the tests :)

sprain commented:
> Tell me if you need help

Thanks! I am currently busy with other things. I'll get back to this as soon as I can.

sprain commented:
@sdespont I fixed and simplified the handling of placeholders by using the official graphics. Any comments by you?
https://github.com/sprain/php-swiss-qr-bill/pull/53/commits/ed95896f5bc1c07709fa0c2fb2ca65e7f66f2bd9#diff-50f34ff1128cf60b1ac0ae7ad32d282b

sprain commented:
@sdespont IMO this is now ready to merge. But I'd be happy to get your review first.

sdespont commented:
@sprain OK for me, you can merge. Thanks for the integration.

sprain commented:
v2.11 it is 😊
https://github.com/sprain/php-swiss-qr-bill/releases/tag/v2.11
